sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1291,8 +1291,9 @@ def _get_previously_launched_zones() -> Optional[List[clouds.Zone]]:
 
             # If it reaches here: the cluster status in the database gets
             # set to either STOPPED or None, since a launch request was issued
-            # but failed, and the provisioner stopped the cluster if
-            # `cluster_ever_up` is True; or terminated the cluster otherwise.
+            # but failed, and the provisioning loop (_retry_zones()) stopped the
+            # cluster if `cluster_ever_up` is True; or terminated the cluster
+            # otherwise.
             if prev_cluster_ever_up:
                 message = (f'Failed to launch the cluster {cluster_name!r}. '
                            'It is now stopped.\n\tTo remove the cluster '
```

Comment:
How about removing `It is now stopped.` and the first `To remove...` hint and only hinting about:
```
To retry launching the cluster, run: sky start {cluster_name}
```

Reply:
Updated. Thanks!